### DISCUSSION TITLE
希望允许自定义每个番剧的目录名

### DISCUSSION THREAD

**Discussion by yonoko** [wushuo894/ani-rss#6] [Ideas]
有些番是已经下载了几集，但是目录名与ani-rss创建的目录名不一样。如果能自定义下载时创建的目录名，就可以比较方便的把以前下载的和当前订阅下载的整合在一起。

**Comment by wushuo894** [comment]
> 有些番是已经下载了几集，但是目录名与ani-rss创建的目录名不一样。如果能自定义下载时创建的目录名，就可以比较方便的把以前下载的和当前订阅下载的整合在一起。

可以做。ani-rss是根据 下载路径+番剧名 拼接成的位置，这个番剧名是你添加订阅时就可以自己修改的。你把下载位置和番剧名都设成和以前一样的不行吗。有自动跳过已下载功能的

**Comment by wushuo894** [comment]
> > 有些番是已经下载了几集，但是目录名与ani-rss创建的目录名不一样。如果能自定义下载时创建的目录名，就可以比较方便的把以前下载的和当前订阅下载的整合在一起。
> 
> 可以做。ani-rss是根据 下载路径+番剧名 拼接成的位置，这个番剧名是你添加订阅时就可以自己修改的。你把下载位置和番剧名都设成和以前一样的不行吗。有自动跳过已下载功能的

https://github.com/wushuo894/ani-rss/blob/master/DOCS.md#%E8%87%AA%E5%8A%A8%E8%B7%B3%E8%BF%87

**Comment by wushuo894** [comment]
以支持
